apps/dapp/src/constants/env/local.tsx

```diff
@@ -1,12 +1,36 @@
 import { Environment } from './types';
 
 const env: Environment = {
+  alchemyId: 'VvVv_fBIiRSaTQzL9RQNybD5FSNvtK5c',
+  backendUrl: 'http://localhost:3001',
+  contracts: {
+    exitQueue: '0x0165878A594ca255338adfa4d48449f69242Eb8F',
```

Comment:
I'd expect this to no longer be used/required by the DApp (IMHO I think this should be tracked as an issue, and hopefully someone in the community can pick it up as a low priority/nice starter task).